docs/DeveloperGuide.md

```diff
@@ -238,6 +238,14 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### Find by address feature
+
+#### Implementation
+
+The find by address/find_add feature filters the list of contacts in the address book based on their address.
+
+Given below is an example usage scenario and how the find_add feature works.
```

Comment:
I want to ask what this statement refers to. Does this refer to the User Story and the Use Case?

Reply:
LGTM